arrow/locales.py

```diff
@@ -78,6 +78,37 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
+    def delocale(self, humanized_str):
+        """consumes:
```

Comment:
Let's change this docstring to conform to the style of the other doc strings in the library.